dGame/dGameMessages/GameMessages.cpp

```diff
@@ -3593,6 +3606,33 @@ void GameMessages::SendBouncerActiveStatus(LWOOBJID objectId, bool bActive, cons
 	SEND_PACKET;
 }
 
+void GameMessages::SendRequestClientBounce(const LWOOBJID objectId, const LWOOBJID bounceTargetId, const NiPoint3& bounceTargetPos, const NiPoint3& bouncedObjLinVel, const LWOOBJID& requestSourceId, const bool bAllBounced, const bool bAllowClientOverload, const SystemAddress& sysAddr) {
```

Comment:
A bit of a note: I'm not sure how to test this with multiple players on my own. I can verify it works for one only.

Reply:
you can launch multiple game clients on your computer at once to test this, if you're physically unable to I can test it